src/main/java/nextstep/subway/member/domain/Member.java

```diff
@@ -17,6 +20,8 @@ public class Member extends BaseEntity {
     private String email;
     private String password;
     private Integer age;
+    @Embedded
+    Favorites favorites;
```

Comment:
```suggestion
    private Favorites favorites;
```

접근제한자 조심해주세요.

---

`Member` 는 회원을 나타내는 굉장히 범용적인 역할을 맡고 있습니다.
회원 정보를 가져올 때마다 `Favorite` 정보를 함께 가져올 필요는 없지 않을까요?

`@OneToMany` 의 기본 fetchType이 LAZY라 요청 시에만 가져오겠지만 `Member` 가 굳이 알아야하나 싶기도 합니다.

Reply:
Member에서 favorite를 알고 있을때만 관리가 가능하지 않을까 싶었는데 다른 방법이 있네요 수정하겠습니다!